src/llfs/page_device_entry.hpp

```diff
@@ -0,0 +1,45 @@
+//#=##=##=#==#=#==#===#+==#+==========+==+=+=+=+=+=++=+++=+++++=-++++=-+++++++++++
+//
+// Part of the LLFS Project, under Apache License v2.0.
+// See https://www.apache.org/licenses/LICENSE-2.0 for license information.
+// SPDX short identifier: Apache-2.0
+//
+//+++++++++++-+-+--+----- --- -- -  -  -   -
+
+#pragma once
+#ifndef LLFS_PAGE_DEVICE_ENTRY_HPP
+#define LLFS_PAGE_DEVICE_ENTRY_HPP
+
+#include <llfs/no_outgoing_refs_cache.hpp>
+#include <llfs/page_arena.hpp>
+#include <llfs/page_device_cache.hpp>
+
+namespace llfs {
+/** \brief All the per-PageDevice state for a single device in the storage pool.
```

Comment:
nit: blank line please

Reply:
Blank line should be there in the latest commit... not sure why the change isn't being marked as outdated, hopefully you can see it.